chapter2/scenarios/06_Dwarven_Doors.cfg

```diff
@@ -513,15 +513,15 @@
     {NOTRAIT_UNIT 2 Swordsman 7 7}
     [message]
       type=Swordsman
-      message=_"Are you finished with the armors?"
+      message=_"Are you finished with the armour suits?"
     [/message]
     [message]
       type=Dwarvish Runesmith
       message=_"Yes, I am."
     [/message]
     [message]
       type=Swordsman
-      message=_"Ok let me see. (examining armors) They are very strong and nice."
+      message=_"Ok let me see. (examining armour) They are very strong and nice."
```

Comment:
I checked all your changes ... Well, I did some research and it turns out that you are right. Plural of armor is armor.
About British versus American English, I have no strong opinion. Sometimes I use American form (e. g. armor), in other cases British (realise).